GAE/test/org/akvo/flow/xml/FlowXmlObjectWriterTests.java

```diff
@@ -513,4 +515,76 @@ void testSerialiseFormWithBarcode() throws IOException {
         //And finally parse it to a DTO
         SurveyDto dto = PublishedForm.parse(xml, true).toDto(); //be strict
     }
+
+    @Test
+    void testSerialiseQuestionLessFormWithTranslations() throws IOException {
+
+        //Mock up a form tree
+        Survey form1 = new Survey();
+        form1.setKey(KeyFactory.createKey("Survey", 17L));
+        form1.setName("This is a form");
+        form1.setVersion(11.0);
+        form1.setSurveyGroupId(123L);
+
+        HashMap<String, Translation> translationHashMap = createTranslationMap("Formulario", "es", form1.getObjectId(),
+                form1.getObjectId(), Translation.ParentType.SURVEY_DESC);
```

Comment:
```suggestion
                form1.getObjectId(), Translation.ParentType.SURVEY_NAME;
```

Perhaps it should be the name instead?